File: src/main/java/com/distribution/modules/api/pojo/vo/ProductTypeVO.java
package com.distribution.modules.api.pojo.vo;

import com.distribution.pojo.tables.pojos.IntegralTutorial;
import lombok.Data;

import java.util.List;

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.api.pojo.vo
 * @Description TODO(描述)
 * @create 2018/7/8-15:17
 */
@Data
public class ProductTypeVO {
    private String id;
    private String prodName;
    private String prodRate;
    private String prodRemark;
    private String prodImg;
    private String prodType;
    private Double exchangePercent;
    private List<IntegralTutorial> tutorials;
}

File: src/main/kotlin/com/distribution/modules/api/pojo/vo/ProductTypeVO.kt
package com.distribution.modules.api.pojo.vo

import com.distribution.modules.integral.entity.IntegralTutorial

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.api.pojo.vo
 
 * @create 2018/7/8-15:17
 */

data class ProductTypeVO (
        var id: String = "",
        var prodName: String = "",
        var prodRate: String = "",
        var prodRemark: String = "",
        var prodImg: String = "",
        var prodType: String = "",
        var exchangePercent: Double = 0.00,
        var tutorials: List<IntegralTutorial> = mutableListOf()
)
